### PR TITLE
Set proper content-type for QR code templates

### PR DESCRIPTION
**Pull request by AliceBilc**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure `mvn -P integration-tests clean verify` runs for the whole project and, if you touched any code for a service in the `services` folder, ensure it can be run with `spring-boot:run`

### CODE REVIEW

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
   /**
    * the checksum of this file.
    */
-  private String checksum = "";
+  private String checksum;
```
Comment (roesslerj) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java:32]: Why did you change this? The way it was ensured avoiding NPEs...

File: services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFileTest.java [hunk 2]
```diff
@@ -2,22 +2,22 @@
...
 class LocalFileTest {
+
   @ParameterizedTest
   @ValueSource(strings = {
       "version",
       "version/v1/configuration/country",
       "version/v1/diagnosis-keys/country",
       "version/v1/diagnosis-keys/country/DE/date",
-      "version/v1/diagnosis-keys/country/DE/date/2020-06-11/hour" })
+      "version/v1/diagnosis-keys/country/DE/date/2020-06-11/hour"})
```
Comment (roesslerj) [inline services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFileTest.java:20]: Seems that your IDE does some format changes here. Not too important, but can be a real pain for bigger PRs.

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java [hunk 3]
```diff
@@ -73,10 +73,10 @@ protected String createS3Key(Path file, Path rootFolder) {
    * Value for the <code>content-type</code> header.
    *
    * @return Either <a href="https://www.iana.org/assignments/media-types/application/zip">zip</a> or
-   *         <a href="https://www.iana.org/assignments/media-types/application/json">json</a>.
+   *     <a href="https://www.iana.org/assignments/media-types/application/json">json</a>.
    */
   public String getContentType() {
-    if (isConfigFile() || isStatisticFile() || isKeyFile()) {
+    if (isConfigFile() || isStatisticFile() || isKeyFile() || isQrPosterTemplate()) {
```
Comment (roesslerj) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java:79]: What the ticket really wanted, was to ensure the content type for QR poster templates. Although is done via the `isQrPosterTemplate` method, is not _explicitly_ and directly tested. Maybe add a test for that?

File: services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFileTest.java [hunk 4]
```diff
@@ -39,10 +39,10 @@ void testGetContentTypeZip(String path) {
   @ParameterizedTest
   @ValueSource(strings = {
       "version/v1/diagnosis-keys/country/DE/date/2020-06-11",
-      "version/v1/diagnosis-keys/country/DE/date/2020-06-11/hour/13" })
+      "version/v1/diagnosis-keys/country/DE/date/2020-06-11/hour/13"})
   void testIsKeyFile(String path) {
     LocalFile test = new LocalIndexFile(Path.of("/root", path, "/index"), Path.of("/root"));
-    assertTrue(test.isKeyFile());
+    Assertions.assertTrue(test.isKeyFile());
```
Comment (roesslerj) [inline services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFileTest.java:45]: I personally am in favour of the `import static` to use assertions directly. But anyway, this is also a formatting thing and not relevant for your PR. :-)

**Review by roesslerj — CHANGES_REQUESTED** [review]

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java (hunk 1 above)
Comment (AliceBilc) [reply services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java:32]: I changed it because it seemed unnecessary, and was a recommendation Intellij gave as well. Now that you mention NPE, makes more sense to have it initialized.

File: services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFileTest.java (hunk 2 above)
Comment (AliceBilc) [reply services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFileTest.java:20]: I think there was a space that was deleted when I have reformatted the code 🤔

File: services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFileTest.java (hunk 4 above)
Comment (AliceBilc) [reply services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFileTest.java:45]: I dont know when I shoul take in consideration what IntelliJ recommends and when not. In this case, I had this worning: Call to 'assertTrue()' from 'Assert' should be replaced with call to method from 'org.junit.jupiter.api.Assertions'

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java (hunk 3 above)
Comment (AliceBilc) [reply services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java:79]: I have just added a new test

**Review by roesslerj — APPROVED** [review]

**Review by ioangut — APPROVED** [review]
